frontend/src/pages/ProfilePage.js

```diff
@@ -0,0 +1,80 @@
+import { Box, Button, TextField, Typography } from '@mui/material'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { changeUserPassword } from '../api/userService'
+
+const ProfilePage = () => {
+  const [newPassword, setNewPassword] = useState('')
+  const navigate = useNavigate()
+
+  // hardcoded value for now, for those doing cookies/ jwt/ context
+  // need to read from there
+  // to test, check atlas to see if there are changes to the hash
+  const uuid = '6311b4362716d8f9d80288bb'
+
+  const handleChangePassword = async () => {
+    if (!newPassword) {
+      // will add in some ui component in the future to reflect
+      // but for now can just check with console.log
+      console.log('Password cannot be empty!')
+      return
+    }
+    try {
+      const res = await changeUserPassword(uuid, newPassword)
+      console.log('success ', res)
```

Comment:
I think u should add a navigate back to home page here after password change is successful